content/feature/Balance Sheet Implementation - Hellhound14.md

```diff
@@ -0,0 +1,40 @@
+|Date|Profit ($)|Loss ($)|Monthly Earnings ($)|Total ($)|
+|----|----------|--------|--------------------|---------|
+|"2018-01-28"|576|543|33|33|
+|"2018-02-28"|675|521|154|187|
+|"2018-03-28"|432|653|-221|-34|
+|"2018-04-28"|765|321|444|410|
+
+
+### Description
+Creating A Table On A SQL Server Can Help You. `How You Ask?` Well….
+Microsoft Softwares Like `MS Excel Cannot Do Things Postgresql Can`. Let’s Say You Wanna Create And Share Your Financial Conditions Depicted Through A Table To Someone Far Away From You, But You Don’t Want To Fuss Over The Work Of Making A Copy Of The File And Sending It Through An E-Mail. You Can Just Create The Same Table In Postgresql Which Will `Automatically Update The Data On It To A Public Server`, Which Can Be Accessed Easily By Anyone With The `Server Name And Password`.

```

Comment:
I'm not sure what the `code blocks` are being used here for, I am not sure they are appropriate here.